api/swagger.yml

```diff
@@ -29,10 +28,6 @@ components:
       type: apiKey
       in: cookie
       name: internal_auth_session
-    oidc_auth:
-      type: apiKey
-      in: cookie
-      name: oidc_auth_session
```

Comment:
we need this, that's the cookie that is verified by lakeFS.

Reply:
Restored!